recipes/soci/all/conanfile.py

```diff
@@ -28,10 +27,9 @@ class SociConan(ConanFile):
         "with_postgresql":  [True, False],
         "with_boost":       [True, False]
```

Comment:
Always end the options/default_options dicts with a comma.
Formatters, such as black, might collapse a dict when it is not ended with a comma.
See https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma
It's not directly applicable here, because the dict is huge. But I think it's nice to know.